Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM alpine:3.6
+ARG bashver=latest
 
-RUN apk --no-cache add bash \
-    && ln -s /opt/bats/bin/bats /usr/sbin/bats
+FROM bash:${bashver}
+
+RUN bash --version
```

Comment:
Rather than baking this into the image, I'm wondering if you can update the `.travis.yml` script to run:

```bash
docker run -it bash:${bashver} --version
```

Reply:
Assuming it does so after building the bats image for that version, that should work fine, yeah.